nautobot/docs/administration/nautobot-server.md

```diff
@@ -336,6 +336,23 @@ spawned uWSGI worker 5 (pid: 112171, cores: 3)
 
 Please see the guide on [Deploying Nautobot](../installation/wsgi.md) for our recommended configuration for use with uWSGI.
 
+### `startplugin`
+
+`nautobot-server startplugin <name> [directory]`
+
+Create a new plugin with `name`.
+
+This command is similar to the django-admin [startapp](https://docs.djangoproject.com/en/dev/ref/django-admin/#startapp) command, but with a default template directory of `nautobot/core/template/plugin_template`. This command assists with creating a basic file structure for beginning development of a new Nautobot plugin.
+
+Without passing in the destination directory, `nautobot-server startplugin` will use your current directory and the `name` argument provided to create a new directory. We recommend providing a directory so that the plugin can be installed or published easily. Here is an example:
+
```

Comment:
In keeping with the other commands, can you please add documentation for the optional arguments here?

e.g. `--name`, `--extension`, `--template`, etc.

Reply:
I started to, but these are all coming from startapp, and better documented there than I could do justice here.  If you still think it should belong here, then I will add it.